app/src/components/common/TmBalance.vue

```diff
@@ -234,7 +236,13 @@ export default {
   },
   computed: {
     ...mapState([`connection`, `session`]),
-    ...mapGetters([`address`, `networks`, `network`, `stakingDenom`]),
+    ...mapGetters([
+      `address`,
+      `networks`,
+      `network`,
```

Comment:
what is `network` if not `currentNetwork`?

Reply:
it is actually `networkId`, only a String. Yes, I know, it is confusing

Reply:
cool. yeah we should change that one day #refactorthefuture